dev/ci/docker/bionic_coq/Dockerfile

```diff
@@ -44,7 +44,8 @@ ENV COMPILER="4.05.0"
 # Common OPAM packages
 ENV BASE_OPAM="zarith.1.10 ocamlfind.1.9.1 ounit2.2.2.3 odoc.1.5.3" \
     CI_OPAM="ocamlgraph.1.8.8 yojson.1.7.0 cppo.1.6.8" \
-    BASE_ONLY_OPAM="dune.2.7.1 elpi.1.13.6 stdlib-shims.0.1.0"
+    BASE_ONLY_OPAM="dune.2.7.1"
+    FLAMBDA_ONLY_OPAM="elpi.1.13.6 stdlib-shims.0.1.0"
```

Comment:
edge only variables should be in the edge ENV command below (line 65 in original file)